app/views/govuk_publishing_components/components/_devolved_nations.html.erb

```diff
@@ -10,6 +10,7 @@
 
   component_helper = GovukPublishingComponents::Presenters::ComponentWrapperHelper.new(local_assigns)
   component_helper.add_class("gem-c-devolved-nations")
+  component_helper.add_aria_attribute({ label: "Nation" })
```

Comment:
One minor suggestion - should we consider adding the label text to the locale file?

Reply:
Good shout, I've just done this 👍